components/elm/src/biogeophys/SoilTemperatureMod.F90

```diff
@@ -943,14 +948,49 @@ subroutine SoilThermProp (bounds,  num_nolakec, filter_nolakec, &
                   endif
                endif
             endif
+            
+            if (use_T_rho_dependent_snowthk) then ! chose which snow thermal conductivity to use 
```

Comment:
```suggestion
            if (use_T_rho_dependent_snowthk) then ! choose which snow thermal conductivity to use 
```

Reply:
Thanks for the feedback, I made the changes and rebased with master.